packages/@aws-cdk/aws-efs/lib/efs-file-system.ts

```diff
@@ -71,6 +71,23 @@ export enum ThroughputMode {
   PROVISIONED = 'provisioned'
 }
 
+/**
+ * EFS Backup policy.
+ *
+ * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-efs-filesystem-backuppolicy.html
+ */
+export enum BackupPolicy {
+  /**
+   * Turns automatic backups on for the file system.
+   */
+  ENABLED = 'ENABLED',
+
+  /**
+   * Turns automatic backups off for the file system.
+   */
+  DISABLED = 'DISABLED'
+}
```

Comment:
what about the other valid values mentioned by the documentation?
> Allowed values: DISABLED | DISABLING | ENABLED | ENABLING

when would a user establish those values? without them, my initial thought was that this should be a `boolean` but it seems like an enum is useful since there are additional modes of operation.

I'm not familiar with how they're used though.

Reply:
I looked at the documentation and was not sure of the use cases for `DISABLING` and `ENABLING`.

I think we should make it a boolean prop to enable the backups.
Thoughts?

Reply:
@ayush987goyal - I'm not familiar enough with Backup but let me investigate and get back to you! it does feel strange to set any other value other than enable/disable